atlas/packages/components/scripts/build-index.js

```diff
@@ -1,17 +0,0 @@
-const fs = require("fs");
```

Comment:
This script got deleted but is still referenced in `README.md` and `package.json`